listen_test.go

```diff
@@ -685,17 +686,20 @@ func TestListenDiscardRepeatedHandshakes(t *testing.T) {
 				break
 			}
 
+			r := req.(*connRequest)
+			fmt.Printf("got req with: %v\n", r.getRequestIdentifier())
```

Comment:
It's a test file, but ... should this debug line should be removed?

Reply:
Will do